apps/admin-gui/src/app/vos/components/applications-dynamic-list/applications-dynamic-list.component.ts

```diff
@@ -139,6 +144,13 @@ export class ApplicationsDynamicListComponent implements OnInit, OnChanges, Afte
   }
 
   ngOnChanges(): void {
+    this.fedColumnsDisplay = [];
+    this.fedColumnsFriendly.forEach((name) =>
+      this.fedColumnsDisplay.push(
+        this.fedAttrs.find((attr) => attr.friendlyName === name)?.displayName || ''
```

Comment:
```suggestion
        this.fedAttrs.find((attr) => attr.friendlyName === name).displayName
```
It should always exist, no?

Reply:
technically, i think it doesn't have to, if the backend configuration changed (e.g. attribute was removed) after the view configuration was set (and validated). but i guess this solution wouldn't save the day